### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by rugwizangoga**
**Hello Microverse Project**

- Created A.hmtl file that has an h1 main title and a reference to its CSS file
- Created an A.css file that colors an h1 tag to my favorite color 
- Added and configured linters to flag programming errors, bugs, stylistic errors, and suspicious constructs in our files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Atest.md
```
Comment (topeogunleye) [inline .gitignore]: - [ ] As written in the project requirements. KIndly add `test.md` not `Atest.md` to your `.gitignore` file.
- [ ] Kindly also add `node_modules` directory to your `.gitignore file.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @rugwizangoga,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .gitignore (hunk 1 above)
Comment (rugwizangoga) [reply .gitignore]: Hello @topeogunleye,
Thank you for your great reviews.

**Review by Reem-lab — APPROVED** [review]
## Hi @rugwizangoga, 👋🏻👋🏻 

Your project is complete! Great job working on the requested changes 🔥🔥🔥🚀🚀💪🏻
 There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
 - Linters checked passed ✅✅
 - Follow Github flow✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**